README.md

```diff
@@ -344,10 +338,6 @@ We propose to add `JSON.parseImmutable` so we can extract a Record/Tuple type ou
 
 The signature of `JSON.parseImmutable` is identical to `JSON.parse` with the only change being in the return type that is now a Record or a Tuple.
 
-## JSON.stringify
```

Comment:
Not sure why this was removed either; it's good to know, even if there's zero spec text corresponding to it.

Reply:
It was already specified somewhere else in the file, so this was a duplicate.